carma_cloud_client/launch/scripts/open_tunnels.sh

```diff
@@ -118,5 +120,7 @@ exec $CMD &> /dev/null  # create the new tunnel
 if [ "$?" -eq 0 ]; then
    if ssh -i $KEY_FILE $REMOTE_USER@$REMOTE_ADDR "sudo lsof -Pi:$REMOTE_PORT -sTCP:LISTEN" >/dev/null; then
         echo "Reverse tunnel is successfully opened!"
+    else
+        echo "Failed to open reverse tunnel"
```

Comment:
I have noticed that a lot of times even though we dont get the successful log, the tunnel still opens. So I am worried this may be confusing.